codegenerator/cli/templates/static/codegen/src/EventFetching.res

```diff
@@ -70,34 +70,6 @@ let makeCombinedEventFilterQuery = (
   })
 }
 
-let applyConditionalFunction = (value: 'a, condition: bool, callback: 'a => 'b) => {
-  condition ? callback(value) : value
-}
-
-let queryEventsWithCombinedFilter = async (
-  ~contractInterfaceManager,
-  ~fromBlock,
-  ~toBlock,
-  ~minFromBlockLogIndex=0,
-  ~provider,
-  ~logger: Pino.t,
-): array<Ethers.log> => {
-  let combinedFilterRes = await makeCombinedEventFilterQuery(
-    ~provider,
-    ~contractInterfaceManager,
-    ~fromBlock,
-    ~toBlock,
-    ~logger,
-  )
-
-  combinedFilterRes->applyConditionalFunction(minFromBlockLogIndex > 0, arrLogs => {
-    arrLogs->Belt.Array.keep(log => {
-      log.blockNumber > fromBlock ||
-        (log.blockNumber == fromBlock && log.logIndex >= minFromBlockLogIndex)
-    })
-  })
-}
```

Comment:
I've noticed that we never set `minFromBlockLogIndex` to something besides `0`, so the filter was never used.